### PR TITLE
add a max-height and overflow for the multiselects

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

File: src/styles/elements/GeoSelect/_geo-multi-select.scss [hunk 1]
```diff
@@ -5,6 +5,8 @@
     display: flex;
     flex-wrap: wrap;
     margin-bottom: -$space-x-small;
+    max-height: 128px;
```
Comment (Navas23) [inline src/styles/elements/GeoSelect/_geo-multi-select.scss:8]: Have we agreed these changes with product?

**Review by Navas23 — CHANGES_REQUESTED** [review]
We are missing the changelog update

File: src/styles/elements/GeoSelect/_geo-multi-select.scss (hunk 1 above)
Comment (irmakyz) [reply src/styles/elements/GeoSelect/_geo-multi-select.scss:8]: - [ ] confirm with product

Comment (irmakyz) [reply src/styles/elements/GeoSelect/_geo-multi-select.scss:8]: i have just confirmed

**Review by dletran — APPROVED** [review]

File: src/styles/elements/GeoSelect/_geo-multi-select.scss (hunk 1 above)
Comment (Navas23) [reply src/styles/elements/GeoSelect/_geo-multi-select.scss:8]: Perfect, add the changes to the changelog then

**Review by Navas23 — CHANGES_REQUESTED** [review]

**Review by dletran — COMMENTED** [review]
Let's not forget about adding an entry to the changelog

**Comment by irmakyz** [comment]
> Let's not forget about adding an entry to the changelog

- [ ] add changelog

**Review by Navas23 — APPROVED** [review]

**Review by fofi — APPROVED** [review]

**Pull request merged**